app/views/drugs/show.html.haml

```diff
@@ -1,44 +1,72 @@
+/ show.html.erb
+/ view for show action in drug controller
+/ rendered in application layout by drug show action
+
+/ similar to show.html.haml, as in it provides the scaffold for the tabs
+/ - shared summary tab partial
+/ - shared interactions tab partial
+/ - shared claims tab partial
+
 =content_for :title, "DGIdb - #@title Drug Record"
 =content_for :header do
-  %h2
-    %span{id: "drug_name"}
-      =@title
-    %small
-      data sources and alternate names 
-      %a(href="/faq#defined")
-        =icon('question-sign')
+	%h2
```

Comment:
The indentation in this file is off (too far indented). The standard is two spaces per indentation level.